xhtml2pdf/context.py

```diff
@@ -383,7 +383,7 @@ def parseExternal(self, cssResourceName):
             result = self.parse(cssFile.getData())
             self.rootPath = oldRootPath
         except Exception as e:
-            print(e)
+            log.error(e)
```

Comment:
```suggestion
            log.exception(e)
```
Since this is a broad except where we don't know the origin of potential errors, I think it makes sense to log the whole stack trace.